### ISSUE TITLE
Consider providing extension `.xpi` to use with `theme_experiment`

### ISSUE THREAD

**Issue by Jaakkonen** [rafaelmardojai/firefox-gnome-theme#353]
Firefox has added option for having user stylesheets in extensions when pref `extensions.experiments.enabled` is enabled.
https://github.com/andreasgrafen/cascade has a minimal example of packaging which uses this to distribute an `.xpi` which provides the theme.

This would prepare firefox-gnome-theme for the experiment to become stable (if it ever becomes) and make the installation easier for people running development versions of Firefox. Currently Firefox requires signatures for extensions and themes for beta and stable versions so only nightly and developer edition versions would work from the official builds.

**Comment by CleoMenezesJr** [comment]
It's pretty great idea. I was thinking of doing something like this, I even saw a workaround before this option was added. If @rafaelmardojai  doesn't do it, I'll be trying to do it, since I already have experience with creating extensions for Firefox.

**Comment by rafaelmardojai** [comment]
> It's pretty great idea. I was thinking of doing something like this, I even saw a workaround before this option was added. If @rafaelmardojai  doesn't do it, I'll be trying to do it, since I already have experience with creating extensions for Firefox.

Yeah, I'm interested in having this, at least providing one when making a release. 

The blocker right now is that the spect seems to work with a single CSS file, so I have to check if imports work, or I need to setup things to merge the CSS sources with some tool.

Also sometimes I have through about using some CSS pre-processor to ease maintenship, so maybe both things can be made in one shot.
